Cubical/Categories/TypesOfCategories/TypeCategory.agda

```diff
@@ -0,0 +1,184 @@
+{-# OPTIONS --cubical --no-import-sorts --postfix-projections --safe #-}
+
+module Cubical.Categories.TypesOfCategories.TypeCategory where
```

Comment:
I didn't realize this PR also contained type categories. @Saizan didn't you experiment with formalizing CwFs at some point?